lib/gollum/public/gollum/javascript/gollum.js.erb

```diff
@@ -47,13 +47,33 @@ $(document).ready(function() {
   $('#delete-link').click( function(e) {
     var ok = confirm($(this).data('confirm'));
     if ( ok ) {
-      var loc = '<%= delete_path %>/' + pageFullPath;
-      window.location = loc;
+      $.post('<%=delete_path %>/' + pageFullPath,
+        {},
+        function (result) {
+          // page successfully deleted, return to landing page
+          window.location = '/';
```

Comment:
@dometto I implemented JS deletion for `/pages` so that the redirect statement in the `/delete` route can be removed with this PR. Cleaner this way. Thoughts?